voidcrew/code/modules/research/techweb/all_nodes.dm

```diff
@@ -22,9 +22,9 @@
 
 /datum/techweb_node/bs_mining
 	id = "bluespace_mining"
-	display_name = "Bluespace Mining Technology"
-	description = "Harness the power of bluespace to make materials out of nothing. Slowly."
-	prereq_ids = list("bluespace_storage", "adv_mining")
+	display_name = "Bluespace Manufacturing Technology"
+	description = "Harness the power of bluespace to make materials out of nothing. And ammo."
+	prereq_ids = list("bluespace_storage", "adv_mining", "adv_weaponry")
 	design_ids = list("bluespace_miner", "miningcore_shell")
```

Comment:
You'll want to add the ammo printer design here so it can be printed

Reply:
I appear to be an idiot